review-of-international-studies.csl

```diff
@@ -0,0 +1,162 @@
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher-place"/>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <citation et-al-min="6" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
+        <text macro="author-short"/>
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+        <group>
+          <label variable="locator" form="short"/>
+          <text variable="locator" prefix=" "/>
+        </group>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" et-al-min="6" et-al-use-first="6">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+    </sort>
+    <layout suffix=".">
+      <text macro="author" suffix="."/>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <group suffix=",">
+            <text macro="title" prefix=" "/>
+            <text macro="editor-translator" prefix=" "/>
+          </group>
+          <text prefix=" " suffix="." macro="publisher"/>
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <text macro="title" prefix=" "/>
+          <group prefix=", ">
+            <text term="in" suffix=": "/>
+            <group delimiter=", " suffix=".">
+              <names variable="editor translator" delimiter=", ">
+                <name and="symbol" sort-separator=", " initialize-with=". "/>
+                <label form="short" prefix=" (" suffix=")" text-case="capitalize-first"/>
+              </names>
+              <group delimiter=", ">
+                <text variable="container-title"/>
+                <group>
+                  <label variable="page" form="short" suffix=" "/>
+                  <text variable="page"/>
+                </group>
+              </group>
+            </group>
+            <text variable="collection-title" prefix=" " suffix="."/>
+            <text macro="publisher" prefix=" " suffix="."/>
+          </group>
+        </else-if>
+        <else>
+          <group suffix=",">
+            <text macro="title" prefix=" "/>
+            <text macro="editor-translator" prefix=" "/>
+          </group>
+          <group prefix=" ">
+            <text variable="container-title"/>
+            <group prefix=", ">
+              <text variable="volume"/>
+              <text variable="issue" prefix=":"/>
+            </group>
+          </group>
+        </else>
+      </choose>
+      <date variable="issued" prefix=" (" suffix=")">
+        <date-part name="year"/>
+      </date>
+      <group prefix=", ">
```

Comment:
pretty sure that will give you the page ranges twice for chapters, here and on line 129 above. Could you check on that?